test/eventbridge_rule_with_sns_fanout/config/targets.yaml

```diff
@@ -0,0 +1,36 @@
+ruby:
+  account_id: 616362385685
+  enabled: false
+php:
+  account_id: 733931915187
+  enabled: false
+dotnetv3:
+  account_id: 441997275833
+  enabled: false
+javascriptv3:
+  account_id: 875008041426
+  enabled: false
+swift:
+  account_id: 637397754108
+  enabled: false
+rust_dev_preview:
+  account_id: 050288538048
+  enabled: false
+cpp:
+  account_id: 770244195820
+  enabled: false
+python:
+  account_id: 664857444588
+  enabled: false
+gov2:
+  account_id: 234521034040
+  enabled: false
+sap-abap:
+  account_id: 099736152523
```

Comment:
These 'numbers' are actually strings, so need "", and this one will be wrong if you don't. (In the block above)